src/odb/src/codeGenerator/schema/chip/dbModBterm.json

```diff
@@ -7,7 +7,7 @@
             "type":"char *",
             "flags":["no-set"],
             "default":"nullptr",
-	    "schema":"db_schema_update_hierarchy"
+	          "schema":"db_schema_update_hierarchy"
```

Comment:
Please don't use tabs in any files as they render differently depending on context